### PR TITLE
Add devise confirmable module to User model

### PR DESCRIPTION
**Pull request by JulianPasquale**
This PR adds confirmable module to User as mentioned in [this card](https://github.com/rootstrap/rails_hotwire_base/projects/1#card-52773599), based on [Devise documentation](https://github.com/heartcombo/devise/wiki/How-To:-Add-:confirmable-to-Users)

### CODE REVIEW

**Review by brunvez — COMMENTED** [review]
Nice, could we add tests?

**Review by brunvez — APPROVED** [review]

**Review by JuannFerrari — APPROVED** [review]
Nice!!!

File: db/schema.rb [hunk 1]
```diff
@@ -50,6 +50,11 @@
     t.datetime "remember_created_at"
     t.datetime "created_at", precision: 6, null: false
     t.datetime "updated_at", precision: 6, null: false
+    t.string "confirmation_token"
+    t.string "unconfirmed_email"
```
Comment (santib) [inline db/schema.rb:54]: Are we using `reconfirmable` ?

**Review by santib — APPROVED** [review]

File: db/schema.rb (hunk 1 above)
Comment (JulianPasquale) [reply db/schema.rb:54]: Based on [this line](https://github.com/rootstrap/rails_hotwire_base/blob/main/config/initializers/devise.rb#L160) I think we are, right?

Comment (santib) [reply db/schema.rb:54]: Seems like it 😄

**Review by horacio — APPROVED** [review]

**Pull request merged**